external_apps/bid/bid_block_func/bid_block_func.go

```diff
@@ -0,0 +1,137 @@
+package bid_block_func
+
+import (
+	"github.com/Oneledger/protocol/action"
+	"github.com/Oneledger/protocol/data/keys"
+	"github.com/Oneledger/protocol/external_apps/bid/bid_action"
+	"github.com/Oneledger/protocol/external_apps/bid/bid_data"
+	"github.com/Oneledger/protocol/external_apps/common"
+	"github.com/google/uuid"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"time"
+)
+
+// Function for block Beginner
+func AddExpireBidTxToQueue(i interface{}) {
```

Comment:
We queued the expired Txs at a block beginner and expire them in the same block's ender?

Reply:
yeah